R/upgrade.R

```diff
@@ -1,21 +1,20 @@
 
 #' Upgrade renv
 #'
-#' Upgrade the version of `renv` associated with a project.
-#'
-#' By default, this function will attempt to install the latest version of
-#' `renv` as available on the active R package repositories. If you'd instead
-#' like to try out a development version of `renv`, you can explicitly request a
-#' different version of `renv` and that version of the package will be
-#' downloaded and installed from GitHub. Use `version = "main"` to install the
-#' latest development version of `renv`, as from the `renv` project's [GitHub
-#' page](https://github.com/rstudio/renv).
+#' Upgrade the version of `renv` associated with a project. [renv::update()]
```

Comment:
Feels like this needs to emphasize that `renv::upgrade()` is used only for upgrading `renv` itself, whereas `renv::update()` is for updating arbitrary packages in a project.